raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -86,7 +86,7 @@ class InitiatorSetup(NamedTuple):
     current_state: State
     block_number: typing.BlockNumber
     channel: NettingChannelState
-    channel_map: typing.ChannelMap
+    channel_map: typing.Dict[typing.ChannelID, NettingChannelState]
```

Comment:
This was moved because of cyclic imports, correct? Isn't it better to have a `ChannelMap`, but defined somewhere else where there are no cycles?

Reply:
The reason is that `ChannelMap` is defined as `Dict[ChannelID, "NettingChannelState"]` with quotes around the type which marshmallow didn't recognize. I decided to just copy the type here instead of medling with the codebase and move alot of things. As you can see i already moved a bunch of other stuff. Maybe we can do this afterwards?